addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -6403,6 +6463,9 @@ registry.ImageTools = ImageHandlerOption.extend({
             const img = this._getImg();
             return isImageSupportedForStyle(img) || this._isImageSupportedForProcessing(img);
         }
+        if (params.name && params.name.startsWith("img_shape_transform_")) {
```

Comment:
Try to avoid doing that, it worsens grepping for no good reason: list all 4 buttons explicitly.
```suggestion
        if ([a, b, c, d].includes(params.name)) {
```